src/pages/Dashboard/TeamOverview.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Button, Center, Flex, Heading, Spacer } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: String!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const StorefrontButton = () => {
+  return <Button w="199px">Share Storefront</Button>;
+};
+
+const SalesInfo = props => {
+  return (
+    <Box mr="100px">
+      <Heading textTransform="uppercase" as="subtitle" size="subtitle" color="gray.500" mb="8px">
+        {props.description}
+      </Heading>
+      <Heading as="h2" size="h2">
+        {props.amount}
+      </Heading>
+    </Box>
+  );
+};
+
+const NoSales = () => {
+  return (
+    <Box w="100%" h="300px" bg="background.primary">
+      <Center w="100%">
+        <Heading as="h4" size="h4" mt="69px" mb="24px">
+          You don’t have sales yet. Share your storelink to get started!
+        </Heading>
+      </Center>
+      <Center w="100%">
+        <StorefrontButton />
+      </Center>
+    </Box>
+  );
+};
+
+const TeamOverview = ({ teamId }) => {
+  const { loading, error, data } = useQuery(GET_TEAM_INFO, {
+    variables: { id: teamId },
+  });
+
+  return loading ? (
+    'Loading...'
+  ) : error ? (
+    `Error! ${error.message}`
+  ) : (
+    <Box w="100%">
+      <Heading textTransform="uppercase" as="subtitle" size="subtitle" color="gray.500" mb="8px">
```

Comment:
heh so I don't think `subtitle` is a real HTML tag. maybe we can render this as an `h2`? or just a `p` tag